### ISSUE TITLE
Calendar Skill sending 'BadRequest' error message

### ISSUE THREAD

**Issue by bensonjm** [microsoft/botframework-components#1332]
## Describe the bug

The Calendar Skill sends a 'BadRequest' message after sending a list of users it found.
This error is shown in the **Emulator**

## Version

Version:  2.1.2
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

## To Reproduce

1. Create the Enterprise Assistant with Calendar and People Skills
2. Use the emulator to test it.
3. Create an event e.g. 'Set up a meeting for tomorrow at 9 AM'
4. Add a title
5. When it asks who it should invite give a name that has multiple entries e.g. John
6. It will return a list of entries for since it's not able to find a unique user
7. See error

## Expected behavior

Emulator should not throw the error

## Screenshots

<img width="683" alt="image" src="https://user-images.githubusercontent.com/48541415/165465697-89b1b484-f2e6-4de7-bd42-4a4c976fa27d.png">

**Comment by ramfattah** [comment]
investigating.

**Comment by ramfattah** [comment]
Hi @bensonjm 

I'm able to reproduce this issue. 

![image](https://user-images.githubusercontent.com/38049078/166076679-fd749c4f-a39a-4e2c-a058-b0b3eeccf463.png)

**Comment by ramfattah** [comment]
Update: please see issue #1333

**Comment by lauren-mills** [comment]
I was able to capture the full error and it looks like this is related to https://github.com/microsoft/botbuilder-dotnet/issues/6307

```
ReasonPhrase:"Bad Request"
Content:"{"errors":{"value.conversation.DialogState.dialogStack[0].state._adaptive.actions[0].dialogStack[0].state._adaptive.actions[0].dialogStack[0].state._adaptive.actions[0].dialogStack[0].state._adaptive.actions[0].dialogStack[0].state.options.choices.value":["The reader's MaxDepth of 32 has been exceeded. Path 'value.conversation.DialogState.dialogStack[0].state._adaptive.actions[0].dialogStack[0].state._adaptive.actions[0].dialogStack[0].state._adaptive.actions[0].dialogStack[0].state._adaptive.actions[0].dialogStack[0].state.options.choices', line 1, position 24486."]},"type":"https://tools.ietf.org/html/rfc7231#section-6.5.1","title":"One or more validation errors occurred.","status":400,"traceId":"|34ab37e9-46a6f1392d5c683f.4.1632613_14.34ab37f2_"}"
```

**Comment by lauren-mills** [comment]
Following up with the SDK team to resolve this at the SDK level.

**Comment by ryub3n** [comment]
@lauren-mills - It's probably related to the SDK but the strange thing is I don't see this error once it's published.

**Comment by EricDahlvang** [comment]
The Bot Builder SDK uses Mewtonsoft v12.  `The reader's MaxDepth of 32 has been exceeded` is from Newtonsoft > v12.

**Comment by ramfattah** [comment]
See also: https://github.com/microsoft/BotFramework-Composer/issues/9208

**Comment by devbynight** [comment]
it seems List card is not supported. see [this](https://github.com/MicrosoftDocs/composer-docs/issues/199) and [this](https://github.com/microsoft/BotFramework-Composer/issues/8933)

**Comment by tracyboehrer** [comment]
The MaxDepth should be resolved in the 4.17 release we published last week.  The default depth is now 128.

**Comment by johnataylor** [comment]
@bensonjm are you still running into this issue with the 4.17 release?
